ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -142,10 +142,10 @@ tests:
     workflow: ipi-vsphere-ovn-hybrid-custom-vxlan-port
 - as: platform-none-vsphere-e2e-operator
   steps:
-    cluster_profile: vsphere
+    cluster_profile: vsphere-platform-none
     test:
-    - ref: windows-e2e-operator-test-with-custom-vxlan-port
-    workflow: upi-vsphere-platform-none-ovn
+    - ref: windows-e2e-operator-test-byoh
+    workflow: windows-e2e-operator-test-platform-none-vsphere
```

Comment:
I was under the impression we were adding a new e2e test job? Here it seems we are replacing the current test in vSphere IPI env (not platform-none) with the new `platform-none` one.

Reply:
You are right, this commit is a follow-up to  147d5ad  were a new e2e test job was added. See https://github.com/openshift/release/commit/147d5ad5cdfc2ca59da4078bfa79d8eb5101f702#diff-be7b1541cba4a85a1c961b8d8a95e4d8706df9a6240e8d9fbc124439485ae999R427

Reply:
We should not be replacing the existing vSphere job

Reply:
Oops I see what happened here, theres no issue then